scripts/ksql/run_ksql_commands.sh

```diff
@@ -3,5 +3,5 @@
 input=/tmp/ksqlcommands
```

Comment:
Should the filename and directory change to `ksqldb/run_ksqldb_commands.sh`? Not sure on this one. Is the query language still called KSQL (instead of ksqlDB)? If so, maybe the filename doesn't need to change.